src/sentry/static/sentry/app/components/events/interfaces/assembly.tsx

```diff
@@ -0,0 +1,108 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styled from 'react-emotion';
+
+import InlineSvg from 'app/components/inlineSvg';
+import TextCopyInput from 'app/views/settings/components/forms/textCopyInput';
+
+interface Props {
+  name: string;
+  version: string;
+  culture: string;
+  publicKeyToken: string;
+  filePath: string;
+}
+
+const Assembly: React.FC<Props> = ({
+  name,
+  version,
+  culture,
+  publicKeyToken,
+  filePath,
+}) => {
+  return (
+    <AssemblyWrapper>
+      <Icon src="icon-return-key" />
+      <AssemblyInfo>
+        <Caption>Assembly:</Caption>
+        {name || '-'}
+      </AssemblyInfo>
+      <AssemblyInfo>
+        <Caption>Version:</Caption>
+        {version || '-'}
+      </AssemblyInfo>
+      <AssemblyInfo>
+        <Caption>Culture:</Caption>
+        {culture || '-'}
+      </AssemblyInfo>
+      <AssemblyInfo>
+        <Caption>PublicKeyToken:</Caption>
+        {publicKeyToken || '-'}
+      </AssemblyInfo>
+
+      {filePath && (
+        <FilePathInfo>
+          <Caption>Path:</Caption>
+          <TextCopyInput>{filePath}</TextCopyInput>
+        </FilePathInfo>
+      )}
+    </AssemblyWrapper>
+  );
+};
+
+// TODO: we should be able to delete these after disabling react/prop-types rule in tsx functional components
```

Comment:
```suggestion
// TODO(ts): we should be able to delete these after disabling react/prop-types rule in tsx functional components
```

The `ts` suffix will help future us find these.

Reply:
thanks, fixed and will keep that in mind in the future